admin/app/dfp/AdUnitAgent.scala

```diff
@@ -41,8 +38,27 @@ object AdUnitAgent extends DataAgent[String, GuAdUnit] {
 
 object AdUnitService {
 
+  // Retrieves the ad unit object if the id matches and the ad unit is active.
```

Comment:
Would it be worth calling this `activeAdUnit` just to be explicit (and complete with `archivedAdUnit` to make it clear that they are non-intersecting sets)?